packages/renderer/src/lib/extensions/InstalledExtensionCardLeftLifecycleStop.svelte

```diff
@@ -16,11 +16,11 @@ async function stopExtension(): Promise<void> {
 }
 </script>
 
-{#if extension.state === 'started' && extension.type !== 'dd'}
+{#if extension.state === 'started'}
   <LoadingIconButton
     clickAction="{() => stopExtension()}"
     action="stop"
     icon="{faStop}"
-    state="{{ status: extension.state, inProgress }}"
+    state="{{ status: extension.type === 'dd' ? 'no-stop' : extension.state, inProgress }}"
```

Comment:
what means the state 'no-stop' ?

Reply:
should it be `unknown` ?

Reply:
I could have used any state other than started (to make the button disabled), but I thought a string like this would appear more intentional for someone looking at the code in the future. Maybe 'stop-unsupported' instead?

Reply:
it's just that inside the LoadingIconButton there is no reference to these states `no-stop` or `stop-unsupported`. So I would stick to some states known by the widget to avoid unexpected behavior (like if someone refactor the component, maybe stop-unsupported will not behave as expected)

`unsupported` might be a new state then rather than `unknown` as we know it's not supported so it should be disabled all the time